app/js/handlebarsUserTable.js

```diff
@@ -11,18 +11,103 @@ function fillUserTable(HBTemplate) {
             return result.json()
         })
         .then(function(result) {
-            let user_list = document.querySelector(".user_list")
-            user_list.innerHTML = ""
-
-            if (result.success) {
-                result.data.forEach(function(userData) {
-                    let html = template(userData)
-                    user_list.innerHTML += html
-                })
-            } else {
-                user_list.innerHTML = "Please contact Admin, user list unavailable"
+            updateDisplayedUsers(result, template)
+        })
+}
+
+/**
+ *this fills the handlebars template with users that have not been soft deleted
+ *
+ * @param apiResponse
+ * @param template
+ */
+function updateDisplayedUsers(apiResponse, template) {
```

Comment:
line 24 function possibly to big? anyway to make multiple functions out of this?